docs/package_info/glossary.rst

```diff
@@ -0,0 +1,46 @@
+Glossary
+========
+
+.. _access_token:
+
+* ``Access Token``: A token that is obtained from authenticating with Reddit.
```

Comment:
To me it's less important how it's obtained but what it's used for. I'd define this something like

> A temporary authentication token used to access the Reddit API. The token lasts…